test/api_get_product_test.dart

```diff
@@ -1430,4 +1432,81 @@ void main() {
       expect(productResult.product!.dataQualityWarningsTags, isNull);
     });
   });
+
+  test('check nutrients modifier (set)', () async {
+    final ProductQueryConfiguration configurations = ProductQueryConfiguration(
+      BARCODE_DANISH_BUTTER_COOKIES,
+      language: OpenFoodFactsLanguage.ENGLISH,
+      fields: [ProductField.ALL],
+      version: ProductQueryVersion.v3,
+    );
+    ProductResultV3 result = await getProductV3InProd(
+      configurations,
+    );
+
+    final Nutriments nutriments = result.product!.nutriments!;
+
+    expect(nutriments.getModifier(Nutrient.energyKJ), isNull);
+    expect(nutriments.getModifier(Nutrient.energyKCal), isNull);
+    expect(nutriments.getModifier(Nutrient.sugars), isNull);
+    expect(nutriments.getModifier(Nutrient.salt), isNull);
+    expect(nutriments.getModifier(Nutrient.fiber), isNull);
+    expect(nutriments.getModifier(Nutrient.fat), isNull);
+    expect(nutriments.getModifier(Nutrient.saturatedFat), isNull);
+    expect(nutriments.getModifier(Nutrient.proteins), isNull);
```

Comment:
Would be more fun if at least one modifier was not null. Or get rid of that test.

Reply:
My goal was to test a product with no modifier.
But I can remove it, no pb